cortex/segment.py

```diff
@@ -46,6 +344,7 @@ def fix_wm(subject):
     subject : str
         Name of the subject to edit
     """
+    warnings.warn("Deprecated! We recommend using edit_segmentation() and rerun_recon() instead of fix_wm() and fix_pia().")
```

Comment:
This raises an error, because `warnings` is not imported

Reply:
Pushed a fix. Sorry for oversight, I don't mind at all if ppl fix trivial little stuff like this on their own initiative.